src/AffectedUserFlow/PublishConsent/PublishConsentForm.spec.tsx

```diff
@@ -4,12 +4,25 @@ import "@testing-library/jest-native/extend-expect"
 import { useNavigation } from "@react-navigation/native"
 
 import PublishConsentForm from "./PublishConsentForm"
-import * as NativeModule from "../../gaen/nativeModule"
 import { Screens } from "../../navigation"
+import { ExposureProvider } from "../../ExposureContext"
 
 afterEach(cleanup)
 
+let mockSubmitDiagnosis: jest.Mock | undefined
```

Comment:
I think ... think that if we expose the raw context and use the factory strategy for this we could avoid having to mock things in advance and we can put these closer to where they are used. Curious to try that experiment.